cf/test-rds.yaml

```diff
@@ -14,6 +14,10 @@ Parameters:
     Type: String
     Default: default
     Description: The sandbox ID of the environment that this cluster is created for.
+  DBInstanceType:
+    Type: String
+    Default: db.t4g.medium
```

Comment:
use the smallest one for testing, we discussed that for the staging envs we should use smaller instances (not only now during development)

Reply:
db.t4g.medium is the smallest instance that we can use with Postgres 13.4. Like the t3, it is only [recommended to use this instance for non-production uses](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/Concepts.DBInstanceClass.html). However, It is [a little bit cheaper](https://aws.amazon.com/rds/postgresql/pricing/) than t3. That's why I use this instance for staging.

Reply:
how inconvenient we didn't notice that when we choose postgre over mysql. could you check how much will it cost to run 1 instance for a day?